Apps/WebClient/src/appsettings.Development.json

```diff
@@ -12,8 +12,8 @@
         "ClientId": "healthgateway-local",
         "RequireHttpsMetadata": "false",
         "Callbacks": {
-            "Logon": "http://localhost:5000/loginCallback",
-            "Logout": "http://localhost:5000/logout"
+            "Logon": "http://0.0.0.0:5000/loginCallback",
```

Comment:
Was this tested?  Generally 0.0.0.0 means bind to all network interfaces but it isn't an actual interface that be be reached.

Reply:
yeah, with that address it binded to the ip address of my pc. I dont know if it will work for mac. but I think it will.

Reply:
so the binding works but you can't hit http://0.0.0.0:5000 or any of the services as its not valid.  I just tested by changing Lab launch settings.  You can hit http://localhost:3004/swagger and https://127.0.0.1/swagger and https://local_ip/swagger but not http://0.0.0.0.  So the changes to the config here I don't think can work for the call back or the endpoints and specifying localhost or anything not your external IP will not work for what we want.  I do think that we should change the launch settings though as it makes it more universal but the endpoints here should remain.